pkg/apis/k0s.k0sproject.io/v1beta1/clusterconfig_types.go

```diff
@@ -365,6 +365,7 @@ func (c *ClusterConfig) GetClusterWideConfig() *ClusterConfig {
 				KubeRouter: c.Spec.Network.KubeRouter,
 				PodCIDR:    c.Spec.Network.PodCIDR,
 				Provider:   c.Spec.Network.Provider,
+				DualStack:  c.Spec.Network.DualStack,
```

Comment:
I wonder if we should make this method live up to its docs. So instead of actively copying stuff, doing it the other way round, deep-copying the input and nilling out the undesired parts.